src/test/librbd/test_librbd.cc

```diff
@@ -2588,6 +2588,10 @@ TEST_F(TestLibRBD, TestScatterGatherIO)
 
   std::string write_buffer("This is a test");
   struct iovec bad_iovs[] = {
+    {.iov_base = &write_buffer[0],  .iov_len = 5},
+    {.iov_base = NULL, .iov_len = static_cast<size_t>(-1)}
+  };
+  struct iovec bad_iovs2[] = {
```

Comment:
I'd flip the names here: having `bad_iovs` consist of one iovec and `bad_iovs2` consist of two iovecs is backwards ;)

Reply:
sure :)